header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -479,7 +479,10 @@ public function define_row_settings( $row_id ) {
 					'partial' => $row_id === 'sidebar' ? 'hfg_header_layout_partial' : $row_setting_id . '_partial',
 				],
 				'options'               => [
-					'priority' => 100,
+					'priority'        => 100,
+					'active_callback' => function () use ( $row_id ) {
+						return $row_id === 'sidebar' || get_theme_mod( 'neve_pro_global_header_settings_advanced_style', true );
+					},
```

Comment:
This will also affect footer rows. We should make sure we check that the current builder is the `header`.

We could probably make it more readable with early returns:

```
if ( builder **is not** header ) 
   return true;

if ( row_id **is** sidebar )
   return true;

if ( get_theme_mod( 'neve_pro_global_header_settings_advanced_style', true ) === true )
  return true;

return false;
```